scss/style.scss

```diff
@@ -28,7 +28,7 @@ $border-divider: 1px solid oColorsGetPaletteColor(pink-tint3);
 	@return $px * 100vh / $vh;
 }
 
-@mixin responsiveType($size, $min: 14px, $max: 0) {
+@mixin responsiveType($size, $min: 17px, $max: 0) {
```

Comment:
20px. The minimum value of `$size`passed in to `responsiveType` is `3vh`, which is added to the `$min` of `17`. I'm gonna refactor this - it's vile.